src/dotnet/Mgmt.CI.BuildTools/NugetToolsPackage/CI.Tools.Package/nuspec/Microsoft.Internal.AzNetSdk.CITools.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
```

Comment:
Is this necessary? .NET tooling should already generate the package for you.

Reply:
This is the only way you can generate the package, but if there are better ways I am open for trying out new things and better ways to do it.
Again can we push this after things are back to normal.